README.md

```diff
@@ -43,21 +43,50 @@ they wish to maintain it going forward.
 See
 [COMMAND_REFERENCE.md](https://github.com/GoogleCloudPlatform/snapshot-debugger/blob/main/snapshot_dbg_cli/COMMAND_REFERENCE.md).
 
-## Download the Snapshot Debugger CLI
+## Installing the Snapshot Debugger CLI
 
-Clone the debugger to your local environment, or to your Cloud Shell
+Install the debugger CLI in your local environment, or in your Cloud Shell
 $HOME directory. See [Using Cloud
 Shell](https://cloud.google.com/shell/docs/using-cloud-shell) for information on
 using Cloud Shell.
 
-Note: When using the Snapshot Debugger in Cloud Shell you will be asked to
+```
+python3 -m pip install snapshot-dbg-cli
+```
+
+> **Note**: When using the Snapshot Debugger in Cloud Shell you will be asked to
 Authorize using your account credentials.
 
-1. Clone the Snapshot Debugger CLI
+
+## Running the Snapshot Debugger CLI
+
+There are two options to run the CLI once the pip install has been completed.
+
+### Option 1: Run the package directly
```

Comment:
Nit: I'd reverse the ordering of these options; the installed script is less typing :)